gradle/javaProject.gradle

```diff
@@ -119,9 +119,9 @@ dependencies {
         implementation 'io.opentracing:opentracing-api:0.33.0'
         implementation 'io.opentracing:opentracing-util:0.33.0'
         implementation 'io.opentracing:opentracing-mock:0.33.0'
-        implementation 'io.swagger.core.v3:swagger-models:1.5.18'
-        implementation 'io.swagger.parser.v3:swagger-parser:2.0.14'
-        implementation 'io.swagger.parser.v3:swagger-parser-v2-converter:2.0.14'
+        implementation "io.swagger.core.v3:swagger-models:2.1.11"
+        implementation "io.swagger.parser.v3:swagger-parser-v2-converter:2.0.28"
+        implementation "io.swagger.parser.v3:swagger-parser:2.0.28"
```

Comment:
Shall we use single quotes in here, to be consistent with the rest of the entries?